addons/dark-www/experimental_profile.css

```diff
@@ -37,3 +37,9 @@
   background-color: #252c37;
   box-shadow: 0 -25px 20px -10px #000;
 }
+.jqui-modal.ui-widget {
+  background: #000;
+}
+.jqui-modal .ui-dialog-content {
+  color: white;
+}
```

Comment:
This was fixed in #1143. It looks like some commits from the 1.6.x branch were never merged into master, but doing that would fix it.

Reply:
I opened a separate PR for this: #1612

Please remove this change.